__tests__/app.test.js

```diff
@@ -34,4 +33,16 @@ describe("GET requests", ()=>{
             .get(`/api/notARoute`)
```

Comment:
This test never failed. If you expect a msg:'Not Found!' then it will fail and you can handle it at the end of your express chain in app.js